### PR TITLE
skip ppc64le builds due to missing upstreams

### PR DESCRIPTION
**Pull request by akrherz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
*  Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

The ppc64le arch is now getting skipped by upstreams like https://github.com/conda-forge/gdal-feedstock/issues/918 , so it needs to be skipped here too...

@conda-forge-admin please rerender

### CODE REVIEW

**Comment by akrherz** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,9 +13,9 @@ source:
   sha256: 1db7f3af66e5dd8b63fb24bde737228c25c7b97a1d40b0597ff4ce1c4b3b0437
 
 build:
-  merge_build_host: true  # [win]
+  merge_build_host: true  # [win or ppc64le]
   skip: true  # [win]
```
Comment (mfansler) [inline recipe/meta.yaml:17]: I believe you wanted:

```suggestion
  merge_build_host: true  # [win]
  skip: true  # [win or ppc64le]
```

**Review by mfansler — CHANGES_REQUESTED** [review]

**Comment by akrherz** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (akrherz) [reply recipe/meta.yaml]: ooopsy :(  Thank you.

**Review by mfansler — APPROVED** [review]
Thanks for submitting these! :heart:

**Comment by davised** [comment]
Looks like the upstreams re-enabled ppc64le.

- https://github.com/conda-forge/gdal-feedstock/pull/968

I'd fiddle with doing a PR for this but I've not managed conda-forge feedstocks before. I was able to re-build locally on ppc64le with the recipe after conversion to rattler-build, so I think that's a success.

**Pull request merged**